src/index.html

```diff
@@ -8,10 +8,189 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <script type="text/javascript" src="main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+    >
+      <fieldset class="blocks">
+        <legend>
+          Personal information
+        </legend>
+        <div class="inputs">
+          <label>
+          Surname:
+            <input
+            type="text"
+            name="Surname"
+            autocomplete="off"
+          >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+            Name:
+            <input
+            type="text"
+            name="Name"
+            autocomplete="off"
+          >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+          How old are You?
+            <input
+            type="number"
+            name="YourAge"
+            min="1" max="100"
+            value="12"
+          >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+            Full date of birth:
+            <input
+            type="date"
+            name="DateOfBirth"
+          >
+          </label>
+        </div>
+        <div>
+          <label>
+            I accept the term of the agreement
+            <input
+            type="checkbox"
+            name="agree"
+          >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="blocks">
+        <legend>
+          Registration
+        </legend>
+        <div class="inputs">
+          <label>
+          E-mail:
+            <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
```

Comment:
```suggestion
            <input
              type="email"
              name="email"
              placeholder="email@example.com"
              required
            >
```
just check your code once again to follow this [style](https://mate-academy.github.io/style-guides/htmlcss.html) 😁